bundles/org.openhab.binding.telegram/src/main/java/org/openhab/binding/telegram/internal/TelegramHandler.java

```diff
@@ -211,13 +212,21 @@ public void initialize() {
                 return UpdatesListener.CONFIRMED_UPDATES_ALL;
         }, exception -> {
                 if (exception != null) {
-                    logger.warn("Telegram exception: {}", exception.getMessage());
+                    logger.debug("Telegram exception: {}", exception.getMessage());

```

Comment:
I don't know if we should just hide all error messages in the debug level output. I think if something goes wrong, every user should be informed about it. I think the level warning was reasonable. Of course, if there is an error that is repeated by the Telegram server, we should somehow stop that.

Reply:
Yes it might be better.
However yesterday I had an internet outage for about 10 minutes and this flooded the log with more than 5000 entries.

Reply:
Maybe warn the user once and then skip logging if the received exception is the same until the ThingStatus is not ONLINE again?

Reply:
Why don't you set the thing status to OFFLINE with the exception message as additional info? The framework should take care of the sitiuation where you set it more than once and discard further updates with the same status. Thing status change events are also logged by default.